data/widgets/options.xml

```diff
@@ -304,7 +304,9 @@
                    text="@.undo_goto_modified"
                    tooltip="@.undo_goto_modified_tooltip" />
             <check id="undo_allow_nonlinear_history"
-                   text="@.undo_allow_nonlinear_history" />
+                   text="@.undo_allow_nonlinear_history" />
```

Comment:
There were some issues with End-Of-Lines characters (it looks like `options.xml` uses CRLF and this modified line was using only LF). I've fixed this in the final commit.